database/migrations/2013_11_17_054526_add_physical_to_assets.php

```diff
@@ -26,6 +26,6 @@ public function up()
      */
     public function down()
     {
-        $table->dropColumn('physical');
+        // $table->dropColumn('physical');
```

Comment:
Why did we end up commenting this one out?

Reply:
Adding the `physical` column is [commented out](https://github.com/snipe/snipe-it/blob/cc370084d1fd2d49de6b4a308563f1d1088149b8/database/migrations/2013_11_17_054526_add_physical_to_assets.php#L17) in this migration's up method so I commented it out in the down method to match it.

Reply:
Ah, got it, thanks!